content/docs/apps/govcloud.md

```diff
@@ -5,30 +5,14 @@ menu:
 title: GovCloud guide
 ---
 
-cloud.gov currently has two "environments" where orgs can live: the legacy East/West environment (located in AWS East/West) and the main GovCloud environment (located in [AWS GovCloud](https://aws.amazon.com/govcloud-us/)). The cloud.gov [FedRAMP JAB P-ATO]({{< relref "overview/security/fedramp-tracker.md" >}}) only covers the GovCloud environment.
-
-**For new users:** All new sandboxes and orgs are in the GovCloud environment, so if you just started using cloud.gov, follow the GovCloud instructions throughout the cloud.gov documentation.
-
-**For teams migrating to GovCloud:** If you've been using cloud.gov for a while, you and your team may have already migrated your orgs and applications to GovCloud, or you may still be in the process of migrating. This page explains changes to look out for in GovCloud.
-
-### Documentation for East/West and GovCloud
-
-cloud.gov documentation has different instructions for East/West and GovCloud users in many places, especially for [logging into the command line interface and dashboard]({{< relref "docs/getting-started/setup.md" >}}). You'll see the following notation:
-
-{{% eastwest %}}
-East/West-specific information
-{{% /eastwest %}}
-
-{{% govcloud %}}
-GovCloud-specific information
-{{% /govcloud %}}
+cloud.gov has consolidated to one "environment" where orgs can live: GovCloud (located in [AWS GovCloud](https://aws.amazon.com/govcloud-us/)). The cloud.gov [FedRAMP JAB P-ATO]({{< relref "overview/security/fedramp-tracker.md" >}}) only covers the GovCloud environment.
```

Comment:
Do you think we should keep this GovCloud transition page at all? Can we just delete it? I imagine at the time we completely shut down East/West, all of our users will already have been oriented on how to use GovCloud... but I'm open to thoughts on this.